source/features/add-gists-link-to-profile.js

```diff
@@ -0,0 +1,14 @@
+import select from 'select-dom';
+import {h} from 'dom-chef';
+
+export default async () => {
+	const username = document.head.querySelector('meta[property="profile:username"]').attributes.content.value;
+
+	const gistsTab = (
+		<a href={`//gist.github.com/${username}`} class="UnderlineNav-item" role="tab" title="Gists">
```

Comment:
`//` is _deprecated_. Always use `https://`

Reply:
![orly](https://user-images.githubusercontent.com/11544418/34265793-f3b34200-e66e-11e7-9e43-6418aab24854.gif)